src/modules/eventRegister/registration-status/useRegistrationStatus.ts

```diff
@@ -0,0 +1,14 @@
+import { computed, Ref } from "vue";
+export default function useRegistrationStatus(step: Ref<number>) {
+  const step1 = computed(() => {
+    return step.value === 1 || step.value === 2 || step.value === 3;
+  });
+  const step2 = computed(() => {
```

Comment:
naming step1, step2, step3 are not very intuitive. Should we change it to something like: enableStep1, enableStep2, enableStep3?